package.json

```diff
@@ -21,7 +21,7 @@
     "error-polyfill": "^0.1.2",
     "local-storage": "^2.0.0",
     "near-api-js": "^0.45.1",
-    "near-social-vm": "NearSocial/VM#1.1.0",
+    "near-social-vm": "near/nearsocial-vm#v1.2.1",
```

Comment:
this actually needs to not make it through to `main`, its a temporary workaround in develop

Reply:
@mpeterdev Sounds good. I'll open up a new PR with the specific changes I need in prod.